docs/hazmat/primitives/asymmetric/serialization.rst

```diff
@@ -463,9 +463,9 @@ Serialization Formats
     method available on
     :class:`~cryptography.hazmat.primitives.asymmetric.dh.DHParametersWithSerialization`.
 
-    .. attribute:: ASN1
+    .. attribute:: DHParameter
 
-        Abstract Syntax Notation One.
+        ASN1 DH parameters sequence as defined in PKCS3.
```

Comment:
Let's link this. I guess https://www.emc.com/emc-plus/rsa-labs/standards-initiatives/pkcs-3-diffie-hellman-key-agreement-standar.htm is probably as canonical as we're going to get for PKCS3.